content/en/docs/kubernetes/collector/components.md

```diff
@@ -267,7 +267,7 @@ Kubernetes-specific receiver, it is still the de facto solution for collecting
 any logs from Kubernetes.
 
 The Filelog Receiver is composed of Operators that are chained together to
-process a log. Each Operator performs a simple responsibility, such as parsing a
```

Comment:
I don't agree with this one either. The explanation doesn't serve to state that an action a user performs is simple; it's a statement of how responsibility is divided amongst operators in the Filelog Receiver. If you'd prefer a different word, please use another way to describe the nature of the level of responsibility for operators in this context.

Reply:
Does "single" responsibility sound good instead?

Reply:
Yep, that could work.